### PR TITLE
Init

### PR DESCRIPTION
**Pull request by odra**
Initial commit, contains the basic structure of container scripts, container source code and openshift templates.

Instructions can be found in the readme file.

There is no backup code logic yet, this PR is supposed to build the base structure for the actual work which will be handled in other PRs/Tickets.

Note: The required secret objects would be created in the installer so those templates are highly configurable.

### CODE REVIEW

File: image/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Integrealy Linux Container
+
+This folder container the linux container source code that runs a intereatly backup script.
+
+## Building
+
+Build the image:
+
+```
+docker build quay.io/integreatly/backup-container:dev .
```
Comment (matskiv) [inline image/README.md]: You are missing `-t ` option before the NAME:TAG -
```docker build -t quay.io/integreatly/backup-container:dev .```

File: templates/openshift/backup-cronjob-template.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    required: true
+  - name: BACKEND_SECRET_NAME
+    description: Backend secret name to create environment variables from
+    required: true
+  - name: ENCRYPTION_SECRET_NAME
+    description: Encruption secret name to create environment variables from
+    required: true
+  - name: CRON_SCHEDULE
+    description: 'Job schedule in Cron Format [Default is everyday at 2am]'
+    value: '*/1 * * * *'
```
Comment (matskiv) [inline templates/openshift/backup-cronjob-template.yaml:69]: Description above("[Default is everyday at 2am]") doesn't match this value.

**Review by matskiv — APPROVED** [review]
I tried the templates and docker build, it seems to work 👍 

Looks good besides one comment above.

File: image/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,20 @@
+FROM centos:7
```
Comment (maleck13) [inline image/Dockerfile:1]: could we use rhel here instead

Comment (matskiv) [reply image/Dockerfile:1]: I think you need RHEL host with subscription to build it. So we would need to build in private jenkins instance.

**Pull request merged**